### DISCUSSION TITLE
TailwindCSS purge does not work.

### DISCUSSION THREAD

**Discussion by suvrat29** [tailwindlabs/tailwindcss#5265] [Help]
Hello everyone,

I'm having trouble with enabling purge in tailwindcss. Everytime I turn purge on, it removes every single style and produces a very unusable page. Even the buttons are not styled as expected. Any help will be greatly appreciated.

Repository: https://github.com/suvrat29/track-expense/

My `tailwind.conf.js` :-

```
module.exports = {
    prefix: "",
    //purge: {
    //  enabled: process.env.NODE_ENV === "production",
    //  content: [
    //    "./src/**/*.{html,ts}",
    //  ]
    //},
    darkMode: "class", // or 'media' or 'class'
    theme: {
      extend: {},
    },
    variants: {
      extend: {},
    },
    plugins: [require("@tailwindcss/forms"),require("@tailwindcss/typography")],
};
```

**Comment by simonswiss** [comment]
Hey!

Are you able to create a minimal reproduction of the issue, instead of the whole app repo?

**Reply by suvrat29** [reply]
I'll create one and will let you know. Thanks
